css/custom.css

```diff
@@ -2567,6 +2567,11 @@ footer {
     background-position: 0 -13rem;
 }
 
+.download-platforms h2 span.icon-dragonflybsd {
```

Comment:
I will have to edit a version of their logo to put in the sprite sheet. I've found an SVG on their official website so it will get edited and added soon.